west.yml

```
@@ -72,7 +72,7 @@ manifest:
       revision: b52fdbf4b62439be9fab9bb4bae9690a42d2fb14
       path: modules/hal/st
     - name: hal_stm32
-      revision: 50283e0438d882a80fb681907664491c7042c817
+      revision: a8498d3aa79595a3ca51ad2dd341d9f28d8343c0
```

```suggestion
      revision: pull/93/head
```
The interest of this method is to be sure we point on and testing the last PR version.

---

OK, changed to `pull/93/head`